README.rst

```diff
@@ -203,3 +203,16 @@ To enable New Relic monitoring for an environment:
         - newrelic_sysmon
 
 * Be sure ``newrelic`` is in the Python requirements of the project.
+
+
+States
+------
+
+papertrail
+~~~~~~~~~~
+
+Including this state forwards all syslog messages to papertrail. Requires
+two secrets:
```

Comment:
Nitpicks: 

1. Double up the colon to get proper formatting of the values below it.
2. Maybe we could say something like 'Add `papertrail` to the base section of top.sls'. Or maybe even check if those secrets are set, so that this state could be a no-op if they're not set, and we could just add that state to the project template? Just a thought... this nitpick is optional :)